blurb/blurb.py

```diff
@@ -745,19 +745,18 @@ def test(*args):
 
 
 def find_editor():
-    for var in 'GIT_EDITOR', 'EDITOR':
-        editor = os.environ.get(var)
-        if editor is not None:
-            return editor
```

Comment:
blurb is not a "git wrapper".  If we stopped using git we would continue using blurb.